python/tank/bootstrap/manager.py

```diff
@@ -23,6 +23,9 @@ class ToolkitManager(object):
     and installations.
     """
 
+    # Maximum progress step number that can be passed to a progress callback.
+    MAX_PROGRESS_STEP_NUMBER = 8  # "Launching Engine..."
```

Comment:
+1 on this. Also means plugins don't have to call out to `sgtk.bootstrap.ToolkitManager.MAX_PROGRESS_STEP_NUMBER` .